Fixed Setting up a cluster: oc tag doesn't work

mburke5678 opened the pull request.

mburke5678 commented:
@miminar Can you take a look at my change based on https://github.com/openshift/origin/issues/8359 and https://github.com/openshift/openshift-docs/issues/2459 
Is this change still needed and if so, what OCP versions does this effect?

miminar commented:
@mburke5678  thanks! This is right. Unless the pull spec is fully qualified (it contains a registry) or has at least 3 name components, it will be considered as an image stream. 

But I'd probably mention an alternative version:

```
oc tag docker.io/openshift/deployment-example:v2 deployment-example:latest
```

At least as a *note*.

mburke5678 commented:
@miminar PTAL. I added the alternative command as a note and moved the second note down paragraph and removed the NOTE to avoid having two notes so close together.

miminar commented on getting_started/administrators.adoc:
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
+====
+You can also use an alternate version of the command:
+----
+oc tag --source=docker openshift/deployment-example:v2 deployment-example:latest
```
`s!--source=docker openshift/deployment!docker.io/openshift/deployment!`

mburke5678 commented:
@miminar Made the change as suggested. PTAL

miminar commented:
/lgtm

mburke5678 commented:
@jianlinliu Can you suggest someone in QE to review these changes? I apologize if this issue is not in your team's scope.

jianlinliu commented:
@wanghaoran1988 this PR is relative to deployment, could you find someone to review these changes.

wanghaoran1988 commented:
LGFM

mburke5678 commented:
@openshift/team-documentation PTAL

In 3.6, this file diverged from the 3.4 and 3.5 versions; I will make the same changes to that file in a separate PR.

ahardin-rh commented on getting_started/administrators.adoc:
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
+====
+You can also use an alternate version of the command:
+----
+oc tag docker.io/openshift/deployment-example:v2 deployment-example:latest
```
$ before the command

ahardin-rh commented on getting_started/administrators.adoc:
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
 source code from within predefined builder images via the Source-to-Image
 toolchain.
 
+For this procedure, make sure that Docker is able to pull images
```
s/make sure/ensure (just a nit)

ahardin-rh commented:
@mburke5678 Just 2 minor comments from me. Otherwise, looks good!

vikram-redhat commented:
@mburke5678  - the commits aren't squashed. Please make sure to squash commits.

vikram-redhat commented:
@mburke5678 - make sure to squash commits.

mburke5678 commented:
[rev_history]
|xref: getting_started/administrators.adoc#try-it-out[Advanced Installation]
|Changed the `tag` command to mark a new image as being desired for deployment.
%

ahardin-rh commented:
The revision history here needs edited. There's an extra space, it's missing some formatting, and the path is not correct. It should be:
xref:../getting_started/administrators.adoc#getting-started-administrators[Administrators: Setting Up a Cluster]

vikram-redhat commented:
@mburke5678 - this also has multiple commits. Please remember to squash.
